README.rst

```diff
@@ -103,30 +103,49 @@ If you are using the electromagnetics module of SimPEG, please cite:
         doi = "http://dx.doi.org/10.1016/j.cageo.2017.06.018"
     }
 
+Questions
+=========
+
+Please do not create an issue to ask a question. You will get a faster
+response by posting on our Discourse discussion forum:
+https://simpeg.discourse.group/. If you prefer real-time chat,
+you can join our slack group at http://slack.simpeg.xyz.
 
```

Comment:
Maybe phrase it positively (first the way to do it, and at the end what NOT to do)? 

«The fastest way go get a response to your answers is by posting on our Discourse discussion forum: https://simpeg.discourse.group/. If you prefer real-time chat,
you can join our slack group at http://slack.simpeg.xyz. Please do not create an issue to ask a question.»

Reply:
I like it.